relay/messages/execution.go

```diff
@@ -42,7 +42,7 @@ type ExecutionResponse struct {
 	Bundle        string      `json:"bundle"`
 	Status        string      `json:"status"`
 	StatusMessage string      `json:"status_message"`
-	Template      string      `json:"template"`
+	Template      interface{} `json:"template"`
```

Comment:
Did this get changed by mistake? I don't see `Template` referenced elsewhere in the PR.